### ISSUE TITLE
Last app from previous workspace is displayed in focused module when you switch to empty workspace on Hyprland

### ISSUE THREAD

**Issue by kepi** [JakeStanger/ironbar#339]
**Describe the bug**

Just follow up for issue fixed in #337. This happens when I have one empty workspace and some workspace with window. When I'm on workspace with window and switch to empty workspace, there is still title of last app in focused module.

**To Reproduce**
1. start new hyprland session with ironbar with focused module on
2. open any app with title
3. switch to empty workspace

**Expected behavior**

I expect focused module to be empty or dissappear on empty workspace.

**System information:**
 - Distro: Arch Linux
- Compositor: Hyprland
 - Ironbar version: v0.13.0.r89.g08e354e-1
 
**Configuration**

<details><summary>Config</summary>

```yaml
center:
  - type: focused
    icon_size: 16
    truncate:
      mode: start

```
</details>

**Comment by JakeStanger** [comment]
Thanks for reporting, I completely missed that when testing.

I've just pushed a fix up on #340 that hopefully squashes it. I'm pretty sure Wayland should return results in the same order every time, but it does risk a race condition since the label is now cleared every time a window loses focus.

If you get a chance, can you give that branch a test and see if it seems reliable please? I'll be doing the same and if all seems good I'll merge in the next couple of days.

**Comment by kepi** [comment]
Sorry, I had been away from computer most days. Just testing current git and works without any problem, thanks!
